src/fixtures/DefaultSalesChannel.ts

```diff
@@ -303,6 +301,51 @@ export const test = base.extend<NonNullable<unknown>, FixtureTypes>({
                 salesChannel: salesChannel.data,
                 customer: { ...customer.data, password: customerData.password },
                 url: baseUrl,
+                themeSeed,
+            });
+        },
+        { scope: 'worker' },
+    ],
+
+    DefaultStorefront: [
+        async ({ browser, AdminApiContext, DefaultSalesChannel, SalesChannelBaseConfig }, use) => {
+            const { id: uuid } = DefaultSalesChannel.salesChannel;
+            const isSaasInstance = await isSaaSInstance(AdminApiContext);
+
+            await AdminApiContext.post(
+                `./_action/theme/${SalesChannelBaseConfig.defaultThemeId}/assign/${uuid}`
+            );
+
+            base.slow();
+
+            if (isSaasInstance) {
+
+                const tmpContext = await browser.newContext();
+                const tmpPage = await tmpContext.newPage();
+
+                for (let i = 0; i < 100; ++i) {
+                    let latestTimestamp = new Date().toISOString();
+                    const response = await AdminApiContext.get(`./notification/message?limit=10&latestTimestamp=${latestTimestamp}`);
+                    const json = await response.json() as { notifications: [{ message: string }], timestamp: string };
+
+                    if (json.timestamp) {
+                        latestTimestamp = json.timestamp;
+                    }
+
+                    if (json.notifications.find(n => n.message.includes(`Compilation for sales channel ${DefaultSalesChannel.salesChannel.name} completed`))) {
+                        tmpPage.close();
+                        tmpContext.close();
+
+                        break;
+                    }
+
+                    // eslint-disable-next-line playwright/no-wait-for-timeout
+                    await tmpPage.waitForTimeout(1000);
+                }
+            }
+
+            await use({
+                ...DefaultSalesChannel,
```

Comment:
Is there a reason why this is a separate fixture if it exposes the exact same data as the `DefaultSalesChannel`? Could be confusing.

Reply:
It's just the default sales channel + theme compilation. So if you just request the sales channel no theme compilation needs to happen